rpp-charter.md

```diff
@@ -1,95 +1,56 @@
-and the outputs of the [httpapi working group](https://datatracker.ietf.org/wg/httpapi/about/).
+All consensus decisions are conducted on the mailing list.
 
 # Deliverables
 
-This working group will publish on its wiki its operational practices,
-including but not limited to, the adoption of work items, usage of
-tools beyond the IETF’s datatracker, and implementation requirements
-for the promotion of work to the IESG.
-
-This working group will find consensus on a set of core requirements for RPP.
-In alignment with the [IESG's Statement on Support Documents dated 2023-08-24](https://datatracker.ietf.org/doc/statement-iesg-support-documents-in-ietf-working-groups-20230824/),
-these requirements will be publicly available on the working group's wiki.
-Analysis of the functionality included and commonly used in core EPP for domain names, 
-hosts and contacts ([RFC5730](https://datatracker.ietf.org/doc/html/rfc5730),
-[5731](https://datatracker.ietf.org/doc/html/rfc5731), [5732](https://datatracker.ietf.org/doc/html/rfc5732), and
-[5733](https://datatracker.ietf.org/doc/html/rfc5733)) is expected to be created as input material for the requirements. 
-The working group may also consider RPP extensions that are functional equivalents of 
-registered [EPP extensions](https://www.iana.org/assignments/epp-extensions/epp-extensions.xhtml) in
-the construction of requirements.
-
-Once established, these requirements may be changed through explicit working group 
-consensus and, when the chairs determine it to be a major change, in consultation 
-with the group’s responsible Area Director. The finding of consensus for both the 
-initial requirements and any changes will be explicitly noted by the chairs on the 
-working group's mailing list.
-
-After finding consensus regarding requirements, the working group will deliver to the
-IESG documents addressing the following topics in an order determined through agreement
-of the chairs with the responsible Area Director:
-* core architecture, including extension mechanisms
-* specifications necessary to implement the core architecture
-* specifications necessary to implement extensions
-* specifications regarding the provisioning of domain names using RPP
-* mappings between RPP and EPP
-
-The working group may consider work for the provisioning of other types of Internet registries (e.g. IP addresses
-and autonomous system numbers) with consultation from the responsible Area Director.
-
-A written record of acceptance with an explanation of working group scope of I-Ds as working group work items 
-is to be recorded with explicit acknowledgement of the chairs and published on the wiki.
+The RPP WG will work on the following milestones within the scope of the charter:
 
+* core architecture, including extension mechanisms (proposed standard)
+* specification(s) necessary to implement the core architecture and extensions (informational)
```

Comment:
Informational does not seem to match a specification, no?

Reply:
yeah, I see your point... it feels like this can be safely collapsed with the proposed standard above.

Reply:
Sounds reasonable.

Reply:
I eliminated the second deliverable (s), but I do not believe this will have an impact on actual milestones

https://github.com/SIDN/ietf-wg-rpp-charter/pull/50/commits/1e7a0cdb5debcdfa0abdba076bd4e542509e8398

If the goal was to keep the architecture informative, I think this will also work:

```
* core architecture, including extension mechanisms (informative)
* specification(s) regarding the provisioning of domain names using RPP (proposed standard)
* mappings between RPP and EPP (proposed standard)
```

If I am missing something important regarding the distinction between these deliverables as they exist in the document today, please let me know.

Reply:
I think "specification(s) necessary to implement the core architecture and extensions" has to be on a level of proposed standard as well.
The idea here was to have 2 levels of standard documents:
- RPP core, which ist not use-case specific (like RFC 5730 and RFC 5734 are for EPP)
- use-case specific application of the above for domain names (RFC5731-33 in case of EPP)